integrations/nuclei_analyzer/compose.yml

```diff
@@ -0,0 +1,15 @@
+# All additional integrations should be added following this format only.
+
+services:
+  nuclei_analyzer:
+    image: pranjalg1310/nuclei-analyzer:1.2.3
```

Comment:
switch to this: https://hub.docker.com/repository/docker/intelowlproject/intelowl_nuclei_analyzer/general